prometheus/0.7.2/artifacthub-pkg.yml

```diff
@@ -59,4 +59,4 @@ recommendations:
 annotations:
   keptn/kind: observability
   keptn/org: contrib
-  keptn/version: 0.9.0
+  keptn/version: 0.9.0-0.11.3
```

Comment:
The versions detailed in the [Promtheus Service README](https://github.com/keptn-contrib/prometheus-service/blob/master/README.md?plain=1#L32) only go up to Keptn 0.10. Is version 0.11 not in the documentation yet?

Reply:
correct - but I've had positive feedback that it works in this span.